socorro/external/postgresql/product_id_map.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+from socorro.external.postgresql.base import PostgreSQLBase
+
+
+logger = logging.getLogger("webapi")
+
+
+class ProductIDMap(PostgreSQLBase):
```

Comment:
Is it likely someone looking for a class for the `product_productid_map` table wouldn't find this one because this is `ProductIDMap` instead of `ProductProductIDMap`? Do other `PostgreSQLBase` class names mirror the tables they derive from?